clusterctl/examples/vsphere/README.md

```diff
@@ -0,0 +1,22 @@
+# Vsphere Example Files
+## Contents
+*.yaml files - concrete example files that can be used as is.
```

Comment:
did you want this in a list? this and the next line shows up together
might help to check out how the md file is parsed [View] button

Reply:
Done